crates/host/src/oci.rs

```diff
@@ -107,6 +109,7 @@ async fn cache_oci_image(
 /// OCI artifact fetcher
 #[derive(Clone, Debug)]
 pub struct Fetcher {
+    additional_ca_paths: Vec<std::path::PathBuf>,
```

Comment:
Nit: can just be `PathBuf` here, right?

Reply:
Good call, yes it can and should be, thanks!